File: Semestr_6/Aplikacje_Mobilne/Lista_0/W02Java/app/src/main/java/com/example/w02java/MainActivity.java
package com.example.w02java;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;
import kotlin.random.Random;

public class MainActivity extends AppCompatActivity {
    private int tem1 = 0;
    private int tem2 = 0;
    private ImageView cpuChoice;
    private ImageView playerChoice;
    private TextView result;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        result = findViewById(R.id.resoult);
        cpuChoice = findViewById(R.id.cpuChoise);
        playerChoice = findViewById(R.id.yourChoise);
    }

    public int roll(){
        tem1 = Random.Default.nextInt(3);
        switch (tem1){
            case 0:
                cpuChoice.setImageResource(R.drawable.rock);
                break;
            case 1:
                cpuChoice.setImageResource(R.drawable.paper);
                break;
            case 2:
                cpuChoice.setImageResource(R.drawable.scissors);
                break;
        }
        return tem1;
    }

    public void choosePaper(View view) {
        playerChoice.setImageResource(R.drawable.paper);
        tem2 = roll();
        switch(tem2){
            case 0:
                result.setText("You Win !!!");
                break;
            case 1:
                result.setText("Tie");
                break;
            case 2:
                result.setText("You Loose );");
                break;
        }
    }

    public void ChooseSciccors(View view) {
        playerChoice.setImageResource(R.drawable.scissors);
        tem2 = roll();
        switch(tem2){
            case 0:
                result.setText("You Loose );");
                break;
            case 1:
                result.setText("You Win !!!");
                break;
            case 2:
                result.setText("Tie");
                break;
        }
    }

    public void chooseRock(View view) {
        playerChoice.setImageResource(R.drawable.rock);
        tem2 = roll();
        switch(tem2){
            case 0:
                result.setText("Tie");
                break;
            case 1:
                result.setText("You Loose );");
                break;
            case 2:
                result.setText("You Win !!!");
                break;
        }
    }
}

File: Semestr_6/Aplikacje_Mobilne/Lista_0/W02a/app/src/main/java/com/example/w02a/MainActivity.kt
package com.example.w02a

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.ImageView
import android.widget.TextView
import kotlin.random.Random

class MainActivity : AppCompatActivity() {
    var tem1 = 0;
    var tem2 = 0;
    lateinit var cpuChoice: ImageView
    lateinit var playerChoice: ImageView
    lateinit var result: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        result = findViewById<TextView>(R.id.resoult)
        cpuChoice = findViewById<ImageView>(R.id.cpuChoise)
        playerChoice = findViewById<ImageView>(R.id.yourChoise)
    }
    fun roll(): Int{
        tem1 = Random.nextInt(3)
        when (tem1){
            0 -> cpuChoice.setImageResource(R.drawable.rock)
            1 -> cpuChoice.setImageResource(R.drawable.paper)
            2 -> cpuChoice.setImageResource(R.drawable.scissors)
        }
        return tem1
    }
    fun choosePaper(view: View) {
        playerChoice.setImageResource(R.drawable.paper)
        tem2 = roll()
        when(tem2){
            0 -> result.text = "You Win !!!"
            1 -> result.text = "Tie"
            2 -> result.text = "You Loose );"
        }
    }
    fun ChooseSciccors(view: View) {
        playerChoice.setImageResource(R.drawable.scissors)
        tem2 =roll()
        when(tem2){
            0 -> result.text = "You Loose );"
            1 -> result.text = "You Win !!!"
            2 -> result.text = "Tie"
        }
    }
    fun chooseRock(view: View) {
        playerChoice.setImageResource(R.drawable.rock)
        tem2 = roll()
        when(tem2){
            0 -> result.text = "Tie"
            1 -> result.text = "You Loose );"
            2 -> result.text = "You Win !!!"
        }
    }
}